styleguide.md

```diff
@@ -0,0 +1,792 @@
+# Styleguide
```

Comment:
Nobody will read this file. If you want to enforce a certain style, use linter rules and a github plugin which checks these. Devs are lazy.

Reply:
Automated linting is planned for the near future: https://ticketsystem.schul-cloud.org/browse/AN-3

Reply:
Great, so I would recommend to remove this file and introduce automated linting in the future instead.

Reply:
Would it be okay to leave this file for now and remove it once automated linting is added?

Reply:
I would vote for no, but you could escalate the decision to a PO.

Reply:
However, there are non-localized hardcoded strings in this file as well.